src/sentry/static/sentry/app/views/releasesV2/detail/overview/commitAuthorBreakdown.tsx

```diff
@@ -74,14 +86,24 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
     );
 
     // sort authors by number of commits
-    const sortedAuthorsByNumberOfCommits = Object.values(groupedAuthorCommits).sort(
+    let sortedAuthorsByNumberOfCommits = Object.values(groupedAuthorCommits).sort(
       (a, b) => b.commitCount - a.commitCount
     );
 
     if (!sortedAuthorsByNumberOfCommits.length) {
       return null;
     }
 
+    // collapse them
+    const MAX = CommitAuthorBreakdown.MAX_WHEN_COLLAPSED;
+    const initialNumberOfAuthors = sortedAuthorsByNumberOfCommits.length;
+    const canCollapse = initialNumberOfAuthors > MAX;
```

Comment:
`canExpand` might be a better name. :bike::house:

Reply:
Thanks!